### DISCUSSION TITLE
HELP ;)  could not get device by ID

### DISCUSSION THREAD

**Discussion by PieterDCK** [oh2th-homey/fi.oh2th.aprs-is#1] [Q&A]
Hi , 
as HAM radio amateur ( call ON7DCK ) from Belgium I recently received my new Homey PRO 2023 version. 

I'm using APRS (car and portable) and I have a WX station at home ( Garni 940) so I discovered this app and I am trying to figure out how this is working ... 

I found a passcode generator for the APRS-IS network, so I entered my callsign and passcode in the setup, but I always get the error ' could not get device by ID '  so I am stuck there ...

So euhm....help ;)  Any suggestions ?? I tried re-installing , but same issue appears 

73's

Pieter

**Comment by oh2th** [comment]
Could you restart the app on Homey, then add a new WX station with your callsign and passcode and all other information.
Once you get the error, go in the Homey app to More -> Apps -> APRS-IS Client
From the App you can "create diagnostics report"

Also if you can take screen shots of the steps you take to configure.

----

Additionally you would need your weather station connected with Homey Pro to collect the weather data to populate information to the APRS WX virtual device. Looks like there is no direct app for Garni, but you could bring the data with [Weather Underground](https://homey.app/a/com.weatherunderground) app to your Homey and use that data.
